### DISCUSSION TITLE
private css Slideshow

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#267] [Q&A] [answered]
Hi,
would it be possible to adjust the look of the stand alone slide show via privat.css.
If yes what elements on the page .

In general how can I identivy points where I can use the privat.css.

**Comment by mgineer85** [comment, marked as answer]
Hello @Striker62,

you usually use the browser to identify elements to change.
Maybe this guide is helpful for you: https://developer.chrome.com/docs/devtools/css?hl=de

Some concrete examples you find in the documentation: https://photobooth-app.org/reference/customizetheme/
